### ISSUE TITLE
The supported installation method is to *only* run ./install.sh, and run it first. As far as I'm concerned this issue steps outside of normal use cases.

### ISSUE THREAD

**Issue by akeysny** [pwnlandia/mhn#831]
There are issues exactly like the one described, pls reopen. Flask version is up to date and so is honeymap url, after running `sudo ./install.sh` honeymap is not rendering. Tried reinstalling everyth many times.

```geoloc                           RUNNING   pid 26249, uptime 0:07:13
honeymap                         FATAL     can't find command '/opt/honeymap/server/server'
hpfeeds-broker                   RUNNING   pid 7128, uptime 0:08:54
mhn-celery-beat                  RUNNING   pid 28001, uptime 0:00:53
mhn-celery-worker                RUNNING   pid 28090, uptime 0:00:41
mhn-collector                    RUNNING   pid 28003, uptime 0:00:53
mhn-uwsgi                        RUNNING   pid 28006, uptime 0:00:53
mnemosyne                        RUNNING   pid 25473, uptime 0:07:40
```

**Comment by wrharding** [comment]
See #813

**Comment by wrharding** [comment]
This was fixed with pwnlandia/honeymap#2.
